examples/movies/current_live.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"github.com/nobuf/cas"
+	"log"
+	"os"
+)
+
+func main() {
```

Comment:
Maybe not a big deal, but two main() in the directory. How about moving to examples/user_current_live?